src/devrules/tui/services/github_service.py

```
@@ -103,3 +120,180 @@ def is_configured(self) -> bool:
+            for p_key in projects_to_check:
+                try:
+                    owner, p_num = resolve_project_number(p_key)
+                    project_item = find_project_item_for_issue(owner, p_num, issue.number)
+                    if project_item:
+                        issue.status = project_item.status
+                        issue.project_name = p_key
+                        found_in_any_project = True
+                        break
+                except Exception:
+                    # Issue might not be in this project
+                    continue
```

Ruff (0.14.14)

`try`-`except`-`continue` detected, consider logging the exception

(S112)